app/assets/javascripts/blacklight/hierarchy/hierarchy.js

```diff
@@ -2,30 +2,32 @@ Blacklight.onLoad(function(){
   Blacklight.do_hierarchical_facet_expand_contract_behavior();
 });
 
-(function($) {
-  Blacklight.do_hierarchical_facet_expand_contract_behavior = function() {
-    $( Blacklight.do_hierarchical_facet_expand_contract_behavior.selector ).each (
-        Blacklight.hierarchical_facet_expand_contract
-     );
+(function() {
+  Blacklight.do_hierarchical_facet_expand_contract_behavior = () => {
+    const elements = document.querySelectorAll(Blacklight.do_hierarchical_facet_expand_contract_behavior.selector);
```

Comment:
It looks like you dropped line-ending semi-colons above. Could this one also be dropped?